loops_lab.js

```diff
@@ -0,0 +1,313 @@
+// 1. Write a `while` loop and a `for` loop that takes the variable "num" and logs all the numbers, descending, between "num" and 1.
```

Comment:
Good demonstration of concept and implementation of for and while loops. Other than some minor feedbacks below, no major errors. Good job and keep up the good work!